social-network-api/src/main/java/fs/socialnetworkapi/entity/Like.java

```diff
@@ -0,0 +1,32 @@
+package fs.socialnetworkapi.entity;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import jakarta.persistence.Column;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Entity
+@Table(name = "likes")
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+public class Like extends AbstractEntity{
+
+  @ManyToOne
+  @JoinColumn(name = "user_id")
+  private User user;
+
+  @ManyToOne
+  @JoinColumn(name = "post_id")
+  private Post post;
+
+  @Column(name = "is_liked")
+  private boolean isLiked;
```

Comment:
you don't need this field. If the record is present it means that the post is liked, if not it's without like